### DISCUSSION TITLE
Anybody else experiencing issues with actions artifacts?

### DISCUSSION THREAD

**Discussion by s-h-a-d-o-w** [community/community#30401] [Actions]
When trying to download one, I get the following message:
```
{ "message": "GitHub Actions is temporarily unavailable. Please visit https://www.githubstatus.com/ for the status of our services.", "ref": "Ref A: 6A1528DB522F47DDB02068719A197419 Ref B: VIEEDGE2409 Ref C: 2022-08-22T13:49:59Z" }
```

Yet, on GitHub status, everything is green.

**Comment by s-h-a-d-o-w** [comment]
Seems to have been resolved.

**Reply by JamesBremner** [reply]
Not sure this is the same.  I try downloading the artifact by clicking on it ( as suggested by the docs ) but it usually does NOT work.  I can only download it by right clicking and selecting 'save link'
